### PR TITLE
Add DebugServer for fgdbg with support for Websockets

### PR DESCRIPTION
**Pull request by feresr**
Test this by running Filament, and opening a browser to `http://localhost:8082/`
console:
```
var webSocket = new WebSocket('ws://localhost:8082');
webSocket.onmessage = function(data) { console.log(data.data); }
```

### CODE REVIEW

File: filament/src/details/Engine.cpp [hunk 1]
```diff
@@ -646,6 +646,15 @@ int FEngine::loop() {
...
     while (true) {
+        #if FILAMENT_ENABLE_FGDBG
+              if (debug.fgdbg) {
+                static int fgdbgMessage = 0;
+                fgdbgMessage++;
+                if (fgdbgMessage % 100 == 0) {
+                  debug.fgdbg->sendMessage(fgdbgMessage/100);
+                }
+              }
+        #endif
```
Comment (feresr) [inline filament/src/details/Engine.cpp:657]: This is only for testing/debug proposes

File: libs/fgdbg/src/DebugServer.cpp [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+#include "fgdbg/DebugServer.h"
+#include <CivetServer.h>
+#include <utils/Log.h>
+#include <tsl/robin_set.h>
+
+using namespace utils;
+namespace filament::fgdbg {
+
+using namespace utils;
```
Comment (bejado) [inline libs/fgdbg/src/DebugServer.cpp:25]: You have this `using` line twice

Comment (feresr) [reply libs/fgdbg/src/DebugServer.cpp:25]: I **really** want to use it
haha good catch thanks

**Comment by romainguy** [comment]
Can we merge this?

**Comment by romainguy** [comment]
Thanks @feresr!

**Pull request merged**